dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V7.6_2023.10.12__Fix-deleted-primary-bitstreams.sql

```diff
@@ -0,0 +1,34 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+BEGIN;
+
+-- Remove all primary bitstreams that are marked as deleted
+UPDATE bundle
+SET primary_bitstream_id = NULL
+WHERE primary_bitstream_id IN
+    ( SELECT bs.uuid
+           FROM bitstream AS bs
+           INNER JOIN bundle as bl ON bs.uuid = bl.primary_bitstream_id
+           WHERE bs.deleted IS TRUE );
+
+-- Remove all primary bitstreams that don't make part on bundle's bitstreams
```

Comment:
@paulo-graca can you update the comment here? I don't understand what this migration is doing.

I _think_ it is checking for primary bitstream IDs that are not currently part of any bundle (ie, they were marked as primary bitstreams at one point, but are no longer).

Reply:
Yes, that's right. Instead of saying "make part" I meant "belong". I will try to better explain what it does.